certbot/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * The `acme` library can now tell the ACME server to clear contact information by passing an empty
   `tuple` to the `contact` field of a `Registration` message. 
 * Fixed the `*** stack smashing detected ***` error in the Certbot snap on some systems.
+* Added timeout to DNS query function calls for dns-rfc2136 plugin.
```

Comment:
Oh actually, this should be moved up to the new section after merging master.

Reply:
I'll try to, hopefully without messing up the branch.